src/ui/osx/TogglDesktop/test2/AppIconFactory.m

```diff
@@ -0,0 +1,32 @@
+//
+//  AppIconFactory.m
+//  TogglDesktop
+//
+//  Created by Nghia Tran on 11/19/18.
+//  Copyright © 2018 Alari. All rights reserved.
+//
+
+#import "AppIconFactory.h"
+
+@implementation AppIconFactory
+
++ (NSImage *)appIconWithType:(AppIconType)type
+{
+	NSImage *icon;
+
+	switch (type)
+	{
+		case AppIconTypeActive :
+			icon = [NSImage imageNamed:@"AppIconActive"];
+			break;
+		case AppIconTypeDefault :
+			icon = [NSImage imageNamed:NSImageNameApplicationIcon];
```

Comment:
It's better to use `NSImageNameApplicationIcon` rather than "AppIcon", because it will get correct size icon depend on the screen resolution.

If not, the icon quality is pretty bad, because it gets 16px icon on **OS 10.11**.

## Before
![screen shot 2019-01-24 at 10 39 04 am](https://user-images.githubusercontent.com/5878421/51653308-77c61780-1fc5-11e9-97ee-257b745bd2ce.png)

## After
<img width="397" alt="screen shot 2019-01-24 at 10 40 02 am" src="https://user-images.githubusercontent.com/5878421/51653323-814f7f80-1fc5-11e9-8135-c8bb1b8ea051.png">